components/layout_2020/flow/mod.rs

```diff
@@ -446,7 +465,20 @@ fn calculate_inline_content_size_for_block_level_boxes(
         }
     }
 
-    let accumulate = |mut data: AccumulatedData, (size, float, clear)| {
+    impl Default for AccumulatedData {
+        fn default() -> Self {
+            Self {
+                max_size: ContentSizes::zero(),
+                left_floats: ContentSizes::zero(),
+                right_floats: ContentSizes::zero(),
+                depends_on_block_constraints: false,
+            }
+        }
+    }
```

Comment:
I think if `ContentSizes` has a `Default` implementation you can just derive `Default` for `AccumulatedData`. If not perhaps you could add `Default` for `ContentSizes`.